File-Formats/DocIO/Open-and-save-Word-document-in-Blazor.md

```diff
@@ -367,8 +367,8 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 By executing the program, you will get the **Word document** as follows.
 
-![Blazor Client-side output Word document](Blazor_Images/Blazor_Output.png)
+![Blazor WASM output Word document](Blazor_Images/Blazor_Output.png)
 
-N> Even though Word library works in client-side, it is recommended to use server-side deployment. Since the client-side deployment increases the application payload size.
+N> Even though Word library works in WASM, it is recommended to use server deployment. Since the WASM deployment increases the application payload size.
```

Comment:
Even though Word library works in WASM app, it is recommended to use server deployment. Since the WASM app deployment increases the application payload size.